packages/use-contractkit/src/screens/wallet-connect-custom.tsx

```diff
@@ -5,23 +5,26 @@ import Loader from 'react-loader-spinner';
 
 import { CopyText } from '../components';
 import { useWalletConnectConnector } from '../connectors/useWalletConnectConnector';
-import { WalletIds } from '../constants';
-import { Connector } from '../types';
+import { Connector, WalletEntry } from '../types';
 
 interface Props {
   onSubmit: (connector: Connector) => void;
+  wallet: WalletEntry;
 }
 
 const getDeepLink = (uri: string) => {
   return `celo://wallet/wc?uri=${uri}`;
```

Comment:
@dckesler is the deeplink constant or should I look in the registry if it's available?

Reply:
hmmm that's really unfortunate, after some digging there is 2 schemes for mobile
CeloWallet: `celowallet://wc?...`
and Valora/CeloDance: `celo://wallet/wc?...`

Should I get in  touch with CeloWallet to see if we can harmonize that?

Reply:
Maybe eventually but may take a few weeks.

Note the deviation was intentional. Celo Wallet and Celo Terminal use their own URL schemes so they don't interfere with each other. Some users also have both installed.

But even if deep linking were totally standardized, I would still prefer to keep the custom Celo Wallet screen in place because the user needs to choose their platform (web or desktop) for the link to work!